driver/mysql/mysql_test.go

```diff
@@ -29,6 +29,22 @@ func TestMigrate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	migrate(t, driverUrl)
+
+	if _, err := connection.Exec(`DROP TABLE IF EXISTS yolo, yolo1, ` + tableName); err != nil {
```

Comment:
Could you point me where this `yolo` is created?
I see only `drop table` statement in this PR.

We need to find another name if it's used. If I'm confused, other devs will be confused as well.

Thanks